### PR TITLE
FossGraph. Create FossGraphCollectionView

### LINKED ISSUE #1538: Rename ```filters``` to ```filter```
btw, why we use `Filters` as plural? it's a single object. Maybe rename everything?
It can be done in next PR

_Originally posted by @nulls in https://github.com/saveourtool/save-cloud/pull/1505#discussion_r1035980262_

### LINKED ISSUE #1934: Security vulnerabilities: support the view with all vulnerabilities (and search by them)
Now we have a view with a particular security issue, need to have a view with all issues

### PR DESCRIPTION
**Pull request by Cheshiriks**
* Added view for list of vulnerabilities
* Renamed all Filters to Filter in project

Closes #1934
Closes #1538

### CODE REVIEW

File: save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.filters
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property prefixName
+ */
+@Serializable
+data class VulnerabilityFilters(
```
Comment (nulls) [inline save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt:9]: can we rename it to `VulnerabilityFilters`? It's a single filter

Comment (Cheshiriks) [reply save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt]: Yes, sure

Comment (sanyavertolet) [reply save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt]: Suggestion to leave as is (VulnerabilityFilters) in order to keep consistence

Comment (nulls) [reply save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt:9]: we have a task to rename all `Filters` to `Filter` -- no need to produce more

Comment (Cheshiriks) [reply save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt]: Renamed all Filters to Filter in project

**Pull request merged**